.github/workflows/checks.yml

```diff
@@ -160,6 +160,8 @@ jobs:
       fail-fast: false
       matrix:
         python-version: [ "3.8", "3.9", "3.10", "3.11", "3.12" ]
+    env:
+      SEPOLIA_RPC_URL: ${{ secrets.SEPOLIA_RPC_URL }}
```

Comment:
We should update the [development guide](https://starknetpy--1393.org.readthedocs.build/en/1393/development.html#environment-variables) to mention that the newly added tests also require the `SEPOLIA_RPC_URL` to be set. We could also consider extracting these tests into a separate comment (like poe `test_ci_devnet_utils`) to clearly distinguish between tests for core functionalities and devnet utils tests. These changes can be implemented in a separate PR.